docs/protocol-methods.rst

```diff
@@ -1055,3 +1055,178 @@ Identify the client to the server and negotiate the protocol version.
 
   ["ElectrumX 1.2.1", "1.2"]
   "ElectrumX 1.2.1"
+
+masternode.announce.broadcast
+--------------
+
+Pass through the masternode announce message to be broadcast by the daemon.
+
+**Signature**
+
+  .. function:: masternode.announce.broadcast(masternode_announce_message)
+  
+**Result**
+
+  A message with the result of the broadcasting, it could be a success message or an error one including the reason of failure.
```

Comment:
Results are never errors.  Just describe what is returned on success as this is intended to document the "result" item in the dictionary; any RPC can theoretically return an error.